NitroxPatcher/Patches/Dynamic/BreakableResource_SpawnResourceFromPrefab_Patch.cs

```diff
@@ -0,0 +1,59 @@
+using HarmonyLib;
+using Mono.Cecil.Cil;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using NitroxModel.Helper;
+using NitroxPatcher.PatternMatching;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+using static System.Reflection.Emit.OpCodes;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Synchronizes entities that can be broken and that will drop material, such as limestones...
+/// </summary>
+public class BreakableResource_SpawnResourceFromPrefab_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Just as for SpawnOnKill_OnKill_Patch, you need to create a test class. It will also help you debug the transpiler without launching your game.